lib/ts/constants.ts

```diff
@@ -15,3 +15,4 @@
 
 export const HEADER_RID = "rid";
 export const HEADER_FDI = "fdi-version";
+export const API_VERSION_FILE_PATH = "/tmp/apiversion.txt";
```

Comment:
Supposed to be `temp` not `tmp`

Reply:
Change the name of the file to `supertokens-apiversion.txt` and same for the other one

Reply:
> Change the name of the file to supertokens-apiversion.txt and same for the other one

done